### ISSUE TITLE
Encrypted load

### ISSUE THREAD

**Issue by tomsej** [llibra/embulk-output-s3#12]
Can you please add support for encrypted load in this plugin? Thank you

**Comment by llibra** [comment]
Do you mean [SSE](http://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html)?

**Comment by tomsej** [comment]
Yes, or maybe [CSE](http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingClientSideEncryption.html) is better option.

**Comment by dmikurube** [comment]
We'll revisit this after this plugin catches up with Embulk v0.10 API/SPI.
